babel.config.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
I've never seen a `babel.config.json` file before: was `babel.config.cjs` not compatible?

Reply:
It didn't need to be dynamic anymore and [Babel strongly recommends using a `.json` file now](https://babeljs.io/docs/en/config-files#supported-file-extensions).

> We recommend using this file type wherever possible: JS config files are handy if you have complex configuration that is conditional or otherwise computed at build time. However, the downside is that JS configs are less statically analyzable, and therefore have negative effects on cacheability, linting, IDE autocomplete, etc. Since babel.config.json and .babelrc.json are static JSON files, it allows other tools that use Babel such as bundlers to cache the results of Babel safely, which can be a huge build performance win.

Reply:
Just noticed `bundlewatch` is `json` too. Could we prefer `.js` unless it is not an option? I think it has become the standard for config files, with even `eslint` listing it as the first recommended format (amongst its many formats).